apps/microsoft-teams/app-actions/src/services/msteams-bot-service.ts

```diff
@@ -5,11 +5,34 @@ import {
   MsTeamsBotServiceResponse,
   TeamInstallation,
   TestMessage,
+  WorkflowUpdateMessage,
 } from '../types';
 
 export class MsTeamsBotService {
   constructor(public readonly botServiceUrl: string, public readonly apiKey: string) {}
 
```

Comment:
fyi: This method is pretty much an exact copy/pasta from `sendEntryActivityMessage()` the only real difference is the URL.